aspnetcore/client-side/spa/angular.md

```diff
@@ -14,98 +14,15 @@ uid: spa/angular
 
 :::moniker range=">= aspnetcore-8.0"
 
-The ASP.NET Core with Angular project template provides a convenient starting point for ASP.NET Core apps using Angular and the Angular CLI to implement a rich, client-side user interface (UI).
+[!INCLUDE [spa-templates-recommend-vs](../../includes/spa-templates-recommend-vs.md)]
 
-The project template is equivalent to creating both an ASP.NET Core project to act as a web API and an Angular CLI project to act as a UI. This project combination offers the convenience of hosting both projects in a single ASP.NET Core project that can be built and published as a single unit.
+## Visual Studio tutorial
 
-The project template isn't meant for server-side rendering (SSR).
+To get started with an Angular project, follow the [Create an ASP.NET Core app with Angular](/visualstudio/javascript/tutorial-asp-net-core-with-angular) tutorial in the Visual Studio documentation.
```

Comment:
I think we're better off to have to TOC go directly the VS content. Why keep the old content and not just go directly to VS? I'll leave it to @danroth27

Reply:
Some people will still want to use the old templates, and this way they can get the documentation for them by selecting the 7.0 version of the docs. And they find out about that availability when they are in the 8.0 version of the docs. We have TOC entries that go directly to the Visual Studio tutorials, and they're listed first, and their link text identifies them as VS tutorials.

Reply:
@Rick-Anderson One way to address your concern is to set the file-level moniker for the SPA Overview, Angular, and React articles to < 9.0. Then they will automatically drop out of the TOC anytime someone selects a 9.0 moniker. That gives people time to switch to the new templates without losing TOC access to docs for the old ones, but lets us get the old ones out of the TOC when .NET 9 is selected, without us having to worry about creating orphaned articles by just deleting them from the TOC. We already have the TOC links to the VS tutorials, and they will stay there.

Reply:
I wanted to have a discoverable way in the ASP.NET Core doc set for building SPA based apps. I'm assuming that many ASP.NET Core devs will start in the ASP.NET Core docs and will look there for SPA support. So, as long as we have a way to discover the SPA content from the ASP.NET Core docs and a way for existing users to find the old content, I'm happy.